### DISCUSSION TITLE
ALS Plugin files do not show up in Visual Studio

### DISCUSSION THREAD

**Discussion by chris84** [Sixze/ALS-Refactored#261] [Q&A] [answered]
Hello,

When I install the plugin according to the instructions on the readme.md, the ALS plugin does not show up in Visual Studio. So I followed these instructions:

- Clone the repository to your project's Plugins folder, or download the latest release and extract it to your project's Plugins folder.
- Merge the DefaultEngine.ini and DefaultInput.ini files from the plugin's Config folder with files from your project's Config folder.
- Recompile your project._

So the problem is that I cannot easily access the header/cpp files from within my Visual Studio project.  Currently I have to look on my filesystem to find out the structure of the ALS Refactored plugin and that switching around and manual searching feels very slow.

Am I doing something wrong? Is there a way to show the plugin in Visual Studio?

Best regards,
Chris

**Comment by chris84** [comment, marked as answer]
I noticed that the "Plugins" folder appeared after I added an ALS derived C++ class from the Unreal Engine.  (I added a C++ class derived from AAlsCharacter).

So the issue is resolved.

**Comment by nekwo** [comment]
you need to regenerate or refresh the visual studio files every time you switch up the cpp directory, it's an unreal thing
